core/src/main/scala/com.snowplowanalytics.snowplow.collector.core/Service.scala

```diff
@@ -139,11 +140,13 @@ class Service[F[_]: Sync](
   }
 
   override def preflightResponse(req: Request[F]): F[Response[F]] = Sync[F].pure {
+    val allowedHeaders = (List("Content-Type", "SP-Anonymous") ++ config.cors.allowedCorsHeaders).map(CIString(_))
+    val corsHeaders    = NonEmptyList.fromListUnsafe(allowedHeaders)
```

Comment:
Think we can remove the "unsafe" method.  Which is harmless in this case, but stands out as worrisome when you see it in collector code.

```scala
val corsHeaders = NonEmptyList("Content-Type", "SP-Anonymous" :: config.cors.allowedCorsHeaders).map(CIString(_))
```